content/german/components/sound-sensor.md

```diff
@@ -0,0 +1,68 @@
+gedreht muss das entsprechende akustische Signal lauter sein, um vom Sensor erkannt zu werden. Auf die linke Seite gedreht am Potentiometer
+macht den Sensor viel empfindlicher gegenüber leisen Geräuschen. Zu finden ist das Potentiometer wie auf diesem Bild gezeigt: {{< img
+alt="Potentiometer des Sound Sensor" src="components/sound-sensor-potentiometer.jpg" >}}
+
+## Voraussetzungen
+
+### DIP Switches
+
+Für diese Komponente werden keine spezifischen DIP-Switches benötigt, so dass diese in der Standardkonfiguration belassen werden können:
+
+{{< dip-switches >}}
+
+## Verwendung
+
+Nachfolgend wird die Verwendung der Klasse {{< javadoc class="com.pi4j.crowpi.components.SoundSensorComponent" >}} beschrieben.
+
+### Konstruktoren
+
+| Konstruktor                                                                       | Bemerkung                                                                                                                                                   |
+|:----------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `SoundSensorComponent(com.pi4j.context.Context pi4j)`                             | Initialisiert einen Sound Sensor mit dem Standard-Pin für den CrowPi.                                                                                       |
+| `SoundSensorComponent(com.pi4j.context.Context pi4j, int address, long debounce)` | Initialisiert einen Sound Sensor mit einem benutzerdefinierten Pin. Zusätzlich kann mit `debounce` noch eine Entprellzeit in Mikrosekunden angegeben werden |
+
+### Methoden
+
+| Methode                 | Bemerkung                                                                                                                          |
+|:------------------------|:-----------------------------------------------------------------------------------------------------------------------------------|
+| `boolean isNoisy()`     | Gibt true zurück, wenn aktuell ein Geräusch vom Sensor erkannt wird                                                                |
+| `boolean isSilent()`    | Gibt true zurück, wenn aktuell stille herrscht.                                                                                    |
+| `SoundState getState()` | Gibt den aktuellen Zustand des Sensors in Form des SoundState zurück.                                                              |
+| `void onNoise()`        | Setzt den Event Handler, welcher bei auftretendem Lärm am Sensor aufgerufen werden soll. null deaktiviert diesen Event Listener.   |
+| `void onSilence()`      | Setzt den Event Handler, welcher bei verschwundenem Lärm am Sensor aufgerufen werden soll. null deaktiviert diesen Event Listener. |
```

Comment:
Whoops, these methods should actually have `SimpleEventHandler handler` as a parameter and it seems like that is missing in the touch sensor component as well. Can you add this to both of them?